### DISCUSSION TITLE
[Help]: Swiping fast to skip multiple slides return the next slide

### DISCUSSION THREAD

**Discussion by Kareem987** [davidjerleke/embla-carousel#831] [Help] [answered]
### Summary

i'm encountering unexpected behavior in my carousel component when using multiple sliders per view. When I adjust the basis property to achieve a layout with 33.3333% width for each slider, and attempt to swipe quickly to skip multiple slides (e.g., from slide 1 to slide 4), the carousel snaps back to slide 2.

https://github.com/davidjerleke/embla-carousel/assets/85441322/4acb15af-2ea6-45ad-8166-9e6878887266

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### CodeSandbox example

https://codesandbox.io/p/sandbox/carousel-wmzl49

**Comment by davidjerleke** [comment, marked as answer]
@Kareem987 you can group slides with the option [slidesToScroll](https://www.embla-carousel.com/api/options/#slidestoscroll):

```tsx
const [emblaRef] = useEmblaCarousel({ slidesToScroll: 'auto' })
```

There’s an example with this behavior [here](https://www.embla-carousel.com/examples/predefined/#slides-to-scroll) in the docs.

Alternatively, if you still want to be able to scroll to individual slides, you can use the option [skipSnaps](https://www.embla-carousel.com/api/options/#skipsnaps) instead:

```tsx
const [emblaRef] = useEmblaCarousel({ skipSnaps: true })
```

Best,
David
